CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
-cmake_minimum_required(VERSION 3.13.0)
-project(libeconf VERSION 0.1.0 DESCRIPTION "TODO")
+cmake_minimum_required(VERSION 2.8.5)
+project(libeconf C)
+set(PROJECT_VERSION 0.1.0)
```

Comment:
I'm fine with supporting an older cmake version, but i suggest changing it to:
```
cmake_minimum_required(VERSION 2.8.5)

set(VERSION 0.1.0)
cmake_policy(VERSION ${CMAKE_VERSION})
if(${CMAKE_VERSION} VERSION_LESS "3.0.0")
  project(libeconf C)
  set(PROJECT_VERSION ${VERSION})
else()
  project(libeconf LANGUAGES C VERSION ${VERSION})
endif()
```
since cmake changed their versioning policy with 3.0.0 and according to `cmake --help-policy CMP0048` the 'old' way is deprecated and may be removed in a future version.

Reply:
I've included the suggested change and force-pushed.
The only change to your proposal I made is that I moved cmake_policy() up, right after cmake_minimum_required(); it's not related to setting the version and is more related to the new location in my opinion.